munkipkg

```diff
@@ -639,6 +639,18 @@ def build(project_dir, options):
             if retcode:
                 raise BuildError("Could not remove existing %s" % outputname)
 
+        # remove .DS_Store file from the scripts folder
+        if os.path.exists(os.path.join(build_info['scripts'], ".DS_Store")):
+            display("Removing .DS_Store file from the scripts folder")
+            os.remove(os.path.join(build_info['scripts'], ".DS_Store"))
+
+        # make scripts executable
+        for pkgscript in ("preinstall", "postinstall"):
+            scriptpath = os.path.join(build_info['scripts'], pkgscript)
+            if os.path.exists(scriptpath) and oct(os.stat(scriptpath).st_mode)[-3:] != '755':
```

Comment:
Curious why you decided to enforce 755 instead of just checking for the executable and read bits for the owner as I originally suggested?